pandas/tests/frame/test_misc_api.py

```diff
@@ -61,37 +61,32 @@ def test_join_index(self):
         # left / right
 
         f = self.frame.reindex(columns=['A', 'B'])[:10]
-        f2 = self.frame.reindex(columns=['C', 'D'])
+        f2 = self.frame.reindex(columns=['C', 'D'])[5:][::-1]
```

Comment:
http://pandas-docs.github.io/pandas-docs-travis/indexing.html#ix-indexer-is-deprecated

you are looking at released docs 

ix depreciation is not released yet - it's in 0.20.0